package.json

```
@@ -49,6 +49,6 @@
         "mocha-cli": "^1.0.1",
         "node-red": "^3.0.2",
         "node-red-node-test-helper": "^0.3.0",
-        "sinon": "^13.0.1"
+        "sinon": "^14.0.1"
```

```suggestion
        "sinon": "^14.0.2"
```

Shouldnt this be 14.0.2 like [flowforge package.json](https://github.com/flowforge/flowforge/blob/cd7648da8c85efa6ca96584ba4849e2e2af55540/package.json#L126)?

NOTE: device agent is 14.0.0 and nr-files is 13.0.1

---

And just to note: sinon release as of 1 month ago is `15.0.1`
https://www.npmjs.com/package/sinon

When does it ever end 🤣

---

We don't need to keep up, we just need to be consistent. The ^ would have covered the patch version mismatch, but I've merged the change. This now reduces the number of `node_module` dirs created